frb_rust/src/handler.rs

```diff
@@ -277,15 +285,15 @@ impl Error {
     /// The identifier of the type of error.
     pub fn code(&self) -> &'static str {
```

Comment:
Are we still using Error.code, Error.message any more? Since we have refactored the error system.